mysql/datadog_checks/mysql/mysql.py

```diff
@@ -335,8 +337,7 @@ def _new_query_executor(self, queries):
             track_operation_time=True,
         )
 
-    @property
-    def runtime_queries(self):
+    def _get_runtime_queries(self, db):
```

Comment:
I think this piece could stay as `property`. 
```python
@property
def runtime_queries(self):
```
`_check_innodb_engine_enabled` can be added to `def _check_database_configuration` and to be initialized on check run.

Reply:
as discussed on the call keeping the current version for now